### ISSUE TITLE
Please include signed dlls in github releases

### ISSUE THREAD

**Issue by StephenHodgson** [AzureAD/microsoft-authentication-library-for-dotnet#807]
Please include built and signed dlls in releases.

**Comment by jmprieur** [comment]
Hello @StephenHodgson 
Can you please elaborate on why to add the DLLs?  Aren't the NuGet packages enough?

**Comment by StephenHodgson** [comment]
Can't download nuget packages inside of Unity game engine.

It would just be a lot easier to download and consume without having to create an empty project to download the nuget packages every time there's an update or building from the source each time as well.

**Comment by bgavrilMS** [comment]
Nuget packages are just zip archives, all the DLLs are inside.

**Comment by StephenHodgson** [comment]
Aren't you guys using CI/CD for delivery?

This process could be automated.

**Comment by MarkZuber** [comment]
Yes, we use CI/CD and the output of that process is signed and published nuget packages which is the standard distribution mechanism for .net assemblies.  You shouldn't need to create a new project just to download the nuget package, you can use the nuget.exe command line or the nuget.org website to grab the contents, rename .nupkg to .zip and extract the files where you need them.

See this for an example:  https://stackoverflow.com/questions/14894864/how-to-download-a-nuget-package-without-nuget-exe-or-visual-studio-extension

**Comment by StephenHodgson** [comment]
So you're not open to providing an easy way to download and utilize your library outside of Nuget for customers that may not be able to utilize that distribution path. 

Got it.
